tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -269,9 +276,14 @@ jobs:
   - Linux_C_API_Packaging_GPU_TensorRT_x64
   condition: succeeded()
   steps:
-  - checkout: self
+  - checkout: self                           # due to checkout multiple repos, the root directory is $(Build.SourcesDirectory)/onnxruntime
```

Comment:
What if you do not have this "- checkout: self" line?

Reply:
If we do not have "- checkout: self" which means we only have single "- checkout: onnxruntime-inference-examples".
The root directory will be $(Build.SourcesDirectory) according to the document.
https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/multi-repo-checkout?view=azure-devops#checkout-path

Reply:
Probably not. If you only "- checkout: onnxruntime-inference-examples", I guess they will still checkout two repos, and leave onnxruntime in the default place, so that you don't need to change the build scripts.

Reply:
If onnxruntime will be checked out at the default place (in $(Build.SourcesDirectory)), what will onnxruntime-inference-examples be checked out to? it will either mixed with onnxruntime or there is "onnxruntime-inference-examples" in $(Build.SourcesDirectory)? I think both cases make the $(Build.SourcesDirectory) a little bit mess.

Reply:
The two directories as shown in this Pipelines's log is well-structured and clear. https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=198130&view=logs&j=505ca2b7-596d-550d-8417-9b1519e87977&t=36b5d3f1-3d2b-50c7-9a89-1ead04cb2f3d

Reply:
A "onnxruntime-inference-examples" folder in $(Build.SourcesDirectory). yes, it is a little bit mess. So it is up to you.